src/game_api/script/lua_vm.cpp

```diff
@@ -966,6 +966,9 @@ end
     lua["move_entity"] = move_entity_abs;
     /// Teleport grid entity, the destination should be whole number, this ensures that the collisions will work properly
     lua["move_grid_entity"] = move_grid_entity;
+    /// Destroy the grid entity, and its item entities, removing them from the grid without dropping particles or gold.
+    /// Will also destroy monsters or items that are standing on a linked activefloor or chain, though excludes MASK.PLAYER to prevent crashes
+    lua["destroy_grid_entity"] = destroy_grid_entity;
```

Comment:
activefloors are not grid entities

Reply:
Activefloors can be overlayed on grid floors, like chain blocks...